workers/generic-worker/loopback_video_darwin_test.go

```diff
@@ -0,0 +1,26 @@
+//go:build darwin
+
+package main
+
+import (
+	"testing"
+
+	"github.com/mcuadros/go-defaults"
+)
+
+func TestLoopbackVideoReturnsMalformedPayload(t *testing.T) {
+	setup(t)
+
+	payload := GenericWorkerPayload{
+		Command:    helloGoodbye(),
+		MaxRunTime: 30,
+		Features: FeatureFlags{
+			LoopbackVideo: true,
+		},
+	}
+	defaults.SetDefaults(&payload)
+	td := testTask(t)
+	td.Scopes = append(td.Scopes, "generic-worker:loopback-video")
```

Comment:
could add a comment that is malformed-payload because the feature isn't implemented on macOS?